app/gui2/src/util/ast/__tests__/node.test.ts

```diff
@@ -27,3 +28,35 @@ test.each(['## Documentation only'])("'%s' should not be a node", (line) => {
   const node = nodeFromAst(ast)
   expect(node).toBeUndefined()
 })
+
+test.each([
+  { code: 'operator1', expected: undefined },
+  { code: 'operator1 foo bar', expected: undefined },
+  { code: 'operator1.parse_json', expected: { subject: 'operator1', accesses: ['parse_json'] } },
+  {
+    code: 'operator1.parse_json foo bar',
+    expected: { subject: 'operator1', accesses: ['parse_json'] },
+  },
+  {
+    code: 'operator1.parse_json.length',
+    expected: { subject: 'operator1', accesses: ['parse_json', 'length'] },
+  },
+  {
+    code: 'operator1.parse_json.length foo bar',
+    expected: { subject: 'operator1', accesses: ['parse_json', 'length'] },
+  },
+  { code: 'operator1 + operator2', expected: undefined },
+])('Primary application subject of $code', ({ code, expected }) => {
+  const ast = Ast.Ast.parse(code)
```

Comment:
Please add cases of expressions contains two calls (like `operator1.parse_json operator2.to_json`)